net-p2p/retroshare/retroshare-0.6.6.recipe

```diff
@@ -141,11 +141,12 @@ REQUIRES="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:libavcodec$secondaryArchSuffix
-	devel:libavformat$secondaryArchSuffix
-	devel:libavutil$secondaryArchSuffix
+	ffmpeg${secondaryArchSuffix}_devel
+#	devel:libavcodec$secondaryArchSuffix
+#	devel:libavformat$secondaryArchSuffix
+#	devel:libavutil$secondaryArchSuffix
```

Comment:
On a version bump this should be reconsidered, this version doesn't build with ffmpeg5 or ffmpeg6.